test/alternative_spec.rb

```diff
@@ -0,0 +1,55 @@
+require_relative "../spec/spec_helper.rb"
```

Comment:
in fact it is enough if you use `require_relative "spec_helper.rb"` if you have it in one file. What is reason to have spec and test dir? usually just one is enough